release-notes/5.2/5.2.0-preview5.md

```diff
@@ -0,0 +1,106 @@
+# Release Notes
+
+## [Preview Release 5.2.0-preview5.24024.3] - 2024-01-24
+
+This update brings the below changes over the previous release:
+
+### Added
+
+- Added .NET 8 support [#2230](https://github.com/dotnet/SqlClient/pull/2230)
+- Added explicit version for major .NET version dependencies [#2303](https://github.com/dotnet/SqlClient/pull/2303)
+- Added the ability to generate debugging symbols in a separate package file [#2137](https://github.com/dotnet/SqlClient/pull/2137)
+- Added .NET 6 missing dependency for nuspec [#2317](https://github.com/dotnet/SqlClient/pull/2317)
+
+### Changed
+
+- Improved TdsParserStateObject by reducing code redundancy [#2254](https://github.com/dotnet/SqlClient/pull/2254)
+
+### Fixed
+
+- Fixed wrong dependency for netstandard2.0 [#2305](https://github.com/dotnet/SqlClient/pull/2305)
+- Fixed connection to unsubscribe from transaction completion events before returning it to the connection pool [#2301](https://github.com/dotnet/SqlClient/pull/2301)
+- Fixed Microsoft.IdentityModel.JsonWebTokens version [#2290](https://github.com/dotnet/SqlClient/pull/2290) to address [CVE-2024-21319](https://www.cve.org/CVERecord?id=CVE-2024-21319)
+- Fixed SQL authentication assert in debug mode [#2288](https://github.com/dotnet/SqlClient/pull/2288)
+- Fixed InvalidCastException when reading an Always Encrypted date or time column [#2275](https://github.com/dotnet/SqlClient/pull/2275)
+- Fixed token caching to prevent expired access tokens from being reused in a connection pool [#2273](https://github.com/dotnet/SqlClient/pull/2273)
+
+## Target Platform Support
+
+- .NET Framework 4.6.2+ (Windows x86, Windows x64)
+- .NET 6.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+- .NET Standard 2.0+ (Windows x86, Windows x64, Windows ARM64, Windows ARM, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+- Microsoft.Data.SqlClient.SNI 5.2.0-preview1.23340.1 
+- Azure.Identity 1.10.3
+- Microsoft.Identity.Client 4.56.0
+- Microsoft.IdentityModel.JsonWebTokens 6.35.0
+- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.35.0
```

Comment:
Keeping the identical dependencies in the same order. You can ignore this suggestion and sort other dependencies instead.
```suggestion
- Microsoft.IdentityModel.Protocols.OpenIdConnect 6.35.0
- Microsoft.IdentityModel.JsonWebTokens 6.35.0
```